docs/configuration/mftf.md

```diff
@@ -15,6 +15,83 @@ SELENIUM_HOST=selenium-hub
 BROWSER=chrome
 ```
 
+Of note - `selenium-hub` can be unreliable, and you may have a better experience using `selenium-chrome`.
+
+To use `selenium-chrome`:
+
+If `.warden/warden-env.yml` doesn't exist, create it in the root of your project, and insert the following:
+
+```
+version: "3.5"
+services:
+  selenium-chrome:
+    hostname: ${WARDEN_ENV_NAME}_selenium-chrome
+    image: selenium/standalone-chrome${WARDEN_SELENIUM_DEBUG:-}:3.8.1
+    extra_hosts:
+      - ${TRAEFIK_SUBDOMAIN:-app}.${TRAEFIK_DOMAIN}:${TRAEFIK_ADDRESS:-0.0.0.0}
+    environment:
+      - START_XVFB=false
+```
+
+Afterwards, you'll need to edit the MFTF `.env` configuration file (`dev/tests/acceptance/.env`) and specify the new `SELENIUM_HOST`:
+
+```
+SELENIUM_HOST=selenium-chrome
+```
+
+After doing so, you'll need to re-run the commands: `vendor/bin/mftf build:project` and `vendor/bin/mftf generate:tests`
+
+Some MFTF tests that test uploading images for Products (such as `AdminSimpleProductImagesTest` - runnable with: `vendor/bin/mftf run:test AdminSimpleProductImagesTest`) will fail using the default configuration, due to `nginx` not having a large enough `client_max_body_size`, or the `selenium-chrome` tab crashing because `selenium-chrome` has run out of memory.
+
+The `client_max_body_size` issue will present itself when attempting the `bmp` upload test in `AdminSimpleProductImagesTest`; it can be remedied by increasing `nginx`'s `client_max_body_size`.
+
+To do this, create the file `.warden/nginx/conf.d/custom.conf`, and populate it with:
+
+```
+client_max_body_size 100M;
+```
```

Comment:
This should be embedded in the `/etc/nginx/nginx.conf` within the Nginx image. No reason users should need to set a custom Nginx setting here. Looking at the role I've used for Nginx in production, the default value I've used there has been 200m, so let's go with that.

Reply:
Clarification - when you say "embedded in the nginx.conf within the Nginx image"; do you mean "added to the docker.io image"?

Reply:
I've created the following images using wardenenv/nginx:1.16 and wardenenv/nginx:1.17 as bases that include this change in the nginx.conf files - I've tested them, however I'm not sure what the process is to update the containers in warden. Do they get pulled into the wardenenv/ repo, or is updating the base.nginx.yml file to point at them acceptable?